frontend/src/uploads/UploadContainer.js

```diff
@@ -127,41 +128,52 @@ const UploadContainer = () => {
   return (
     <div className="row">
       <div className="col-12 mr-2">
-        <AlertDialog
-          open={openDialog}
-          title={'Replace existing data?'}
-          dialogue={'Selecting replace will delete all previously uploaded records for this dataset'}
-          cancelText={'Cancel'}
-          handleCancel={handleReplaceDataCancel}
-          confirmText={'Replace existing data'}
-          handleConfirm={handleReplaceDataConfirm}
-        />
-        <Stack direction="column" spacing={2}>
-          <Paper square variant="outlined">
-            <UploadPage
-              alertElement={alertElement}
-              uploadFiles={uploadFiles}
-              datasetList={datasetList}
-              doUpload={doUpload}
-              setDatasetSelected={setDatasetSelected}
-              datasetSelected={datasetSelected}
-              setUploadFiles={setUploadFiles}
-              setReplaceData={setReplaceData}
-              replaceData={replaceData}
-              handleRadioChange={handleRadioChange}
-              downloadSpreadsheet={downloadSpreadsheet}
-              setAlert={setAlert}
+        {alert && alertContent && alertSeverity &&
+          <Alert severity={alertSeverity}>{alertContent.split('\n').map((line, index) => (
+            <React.Fragment key={index}>
+              {line}
+              <br />
+            </React.Fragment>
+          ))}</Alert>
+        }
+        <>
+          {open && (
```

Comment:
The `open &&` is not needed here, as `open` is passed to `AlertDialog`, which will render or not render depending on the value of `open`.